cypress-io/cypress
# fix: Increase contrast on spec header items

* Increase contrast on expandable "Specs" button
* Increase contrast on spec search field label and icon



- Closes #21870 (UNIFY-1667)

### User facing changelog
Increase contrast on inline Spec List header items to improve accessibility

### Additional details
Accessibility testing of v10 release identified potential accessibility issues with header items in expandable spec list header items in CT. 

### Steps to test
1. Open cypress
2. Open test runner
3. Observe increased contrast on expanding "Specs" control in header
4. Expand to show search control, observe increased contrast on icon and search field label

### How has the user experience changed?

**Before**
![before](https://user-images.githubusercontent.com/11482842/171652184-8ba7fd9a-edaf-4f72-97e8-ba4cfa053f48.png)

**After**
![after](https://user-images.githubusercontent.com/11482842/171652221-51d444c6-6ecd-45cc-915f-2bbc576799b9.png)

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
